.github/dependabot.yml

```diff
@@ -0,0 +1,12 @@
+version: 2
+updates:
+  - package-ecosystem: "maven"
+    directory: "/"
+    schedule:
+      interval: "daily"
+      time: "02:00"
+  - package-ecosystem: "github-actions"
```

Comment:
Actions are not enabled here and no action is executed, so why have this ecosystem covered at all?

Reply:
Does someone said that actions can never be enabled here?

Reply:
Will you accept gradle ecosystem as well, as noone said it can never be migrated to gradle then?

Reply:
Enabling actions is about filing a bug to the repo owners should it bring in some benefits - how much time that needs? 10 minutes? Have anyone had cycles to investigate benefits it can bring as well as its disadvantages?  Compare that with effort needed to convert this project, and thus _all_ ee4j projects,  to gradle. Discussion wrt gradle was on the table in the past but _who_ and _when_ would do that work?

Reply:
I used gradle only as example of other __not needed now__ dependabot ecosystem.

Reply:
auto-update of dependencies is good thing to have, some projects are using jenkins for it. These projects may go away from that solution at some point and move to what's suggested here too. It is just **not the right time to make decision ** and merge/work on this now.